students/bbirsoz1/Module_3/mailroom1_corrected.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import sys  # imports go at the top of the file
+
+#Mailroom Part 1
+
+donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "Obama": [99999, 3333],
+}
+
+main_menu_prompt = "\n".join(("Welcome to the Mailroom Charity Menu!",
+                    "Please choose from below options:",
+                    "1 - Send a Thank You",
+                    "2 - Create a Report",
+                    "3 - Quit",
+                    ">>> "))
+
+
+thank_menu_prompt = "\n".join(("Please enter one of the following:",
+                    "1 - Enter a new thank you note",
+                    "2 - List of donors in the database",
+                    "3 - Return to main menu",
+                    '>>> '))
+
+def list_of_donors():
+          print("List of donors in the database:")
+          for donor in donor_db:
+              print(donor)          
+
+def create_a_report():
+    print("Report:")
+    Header = ('Donor Name', 'Total Donation', "Number of Donations", 'Average Donation')
+    print("-"*75)
+    print("{:<20} | {:^24} | {:^19} | {:^15}".format(Header[0], Header[1], Header[2], Header[3]))
+    print("-"*75)
+
+    everything = [ ]
+    #donor_db.sort(key=sort_donation, reverse=True) #do I not need this? I got a AttributeError: 'dict' object has no attribute 'sort'
+    for donor_name, donations in donor_db: #I get an error saying too many values to unpack (expected 2)
+        total_donation = sum(donations)
+        num_of_donation = len(donations)
+        average_donation = int(total_donation / num_of_donation)
+        print("{:<20} | {:>24} | {:^19} | {:>15}".format(donor_name, total_donation, num_of_donation, average_donation))
+    print("-"*75)
+     
+def mail_donor():
+    list_of_donors()
+    current_d = "" #donors currently listed
+    current_d = str(input("Who would you like to mail?:"))
+    if current_d in donor_db:
+        donation_amount = int(input("How many donations have been made so far?: "))
```

Comment:
The wording is a little odd here.  Your asking how many donations have been made which suggests a count of donations yet you are storing in `donation_amount` which would suggest a single donation.

Reply:
Hi Mike,
I fixed the comments you had. 
The main issue I am having is line 42. When I try to make a database, I get an error saying too many values to unpack (expected 2).
What is wrong there?